Source/JavaScriptCore/runtime/VM.h

```diff
@@ -378,6 +378,8 @@ class VM : public ThreadSafeRefCountedWithSuppressingSaferCPPChecking<VM>, publi
     // topEntryFrame.
     // FIXME: This should be a void*, because it might not point to a CallFrame.
     // https://bugs.webkit.org/show_bug.cgi?id=160441
+    // The following two fields are sometimes treated as a pair in assembly code, making usages of the second one implicit.
+    // To find them, look for loadpairq/storepairq of "VM::topCallFrame" in *.asm files.
```

Comment:
This might be a bit verbose for my taste, but I'm not sure. Maybe just the first line?

Reply:
I see now that Mark already commented, we can stick with this